frontend-react/e2e/spec/organization-settings-page.spec.ts

```diff
@@ -1,12 +1,231 @@
 import { expect, test } from "@playwright/test";
+import { readFileSync } from "node:fs";
+import { join } from "node:path";
+import { fileURLToPath } from "node:url";
+import { mockOrganizationSettingsList } from "../mocks/organizations";
 import * as organization from "../pages/organization";
 
-test.use({ storageState: "e2e/.auth/admin.json" });
+const __dirname = fileURLToPath(import.meta.url);
 
-test("Has correct title", async ({ page }) => {
-    await organization.goto(page);
-    await page.locator(".usa-table tbody").waitFor({ state: "visible" });
+test.describe("Admin Organization Settings Page", () => {
+    test.describe("not authenticated", () => {
+        test("redirects to login", async ({ page }) => {
+            await organization.goto(page);
+            await expect(page).toHaveURL("/login");
+        });
+    });
 
-    await expect(page).toHaveURL(/settings/);
-    await expect(page).toHaveTitle(/Admin-Organizations/);
+    test.describe("authenticated receiver", () => {
+        test.use({ storageState: "e2e/.auth/receiver.json" });
+        test("returns Page Not Found", async ({ page }) => {
+            await organization.goto(page);
+            await expect(page).toHaveTitle(/Page Not Found/);
+        });
+    });
+
+    test.describe("authenticated sender", () => {
+        test.use({ storageState: "e2e/.auth/sender.json" });
+        test("returns Page Not Found", async ({ page }) => {
+            await organization.goto(page);
+            await expect(page).toHaveTitle(/Page Not Found/);
+        });
+    });
+
+    test.describe("authenticated admin", () => {
+        test.use({ storageState: "e2e/.auth/admin.json" });
+
+        test("If there is an error, the error is shown on the page", async ({
+            page,
+        }) => {
+            await page.route("/api/settings/organizations", (route) =>
+                route.fulfill({ status: 500 }),
+            );
+            await organization.goto(page);
+            await expect(page.getByText("An error has occurred")).toBeVisible();
+        });
+
+        test.describe("When there is no error", () => {
+            test.beforeEach(async ({ page }) => {
+                await page.route("/api/settings/organizations", (route) =>
```

Comment:
You can us the `API_ORGANIZATIONS` const from the organizations.ts file instead of `/api/settings/organizations`